src/components/navigation/header/header.module.css

```diff
@@ -1,52 +1,81 @@
 .focusOn {
-  position: sticky;
   top: 0;
   overflow: hidden;
   z-index: 9999;
-  background-color: var(--background-bg-dark);
 }
 
 .isOpen {
-  height: 100vh;
+  height: 100%;
   display: flex;
   flex-direction: column;
 
   @media (max-width: 1024px) {
-    background: var(--background-bg-dark);
+    background: var(--primary-bg);
 
     & > header {
-      flex: 1;
       display: flex;
       flex-direction: column;
-      padding: 1rem 2.5rem;
 
       & > nav {
         flex: 1;
         display: flex;
         flex-direction: column;
-        justify-content: flex-end;
       }
     }
   }
 }
 
+.spacer {
+  height: 8rem;
+  width: 100%;
+}
+
 .wrapper {
   display: flex;
-  justify-content: space-between;
+  flex-direction: column;
+  position: fixed;
+  top: 0rem;
+  margin: 1.5rem 0;
+  justify-content: center;
   align-items: center;
-  padding: 16px 24px;
+  min-height: 5rem;
+  padding: 0rem 3rem 0rem 3rem;
+  border-radius: 0.75rem;
+  background: rgba(250, 250, 250, 0.75);
```

Comment:
Replace with our global variables

Reply:
Because we have lower opacity on the background, we have no global variable for the color of the header, but I can make a new variable for the header background so we can still control it from global.css!

Reply:
Fixed in [a401d8e](https://github.com/varianter/variant.no/pull/944/commits/a401d8efa08f1db27892c8091773081b13e964b9)